kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorNetworkIT.java

```diff
@@ -285,9 +288,8 @@ private void emptyBucket(String bucketName) {
     // delete all objects to empty bucket
     ObjectListing objectListing = S3Client.listObjects(bucketName);
```

Comment:
Here I meant if we could use the below instead of the whole first while loop, not within it. 
```
for (S3ObjectSummary s3ObjectSummary : objectListing.getObjectSummaries()) { 
    S3Client.deleteObject(bucketName, s3ObjectSummary.getKey());
}
```